how-to-create-typo-variable-fixer/composer.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "samsonasik/app",
+    "description": "App Demo",
+    "require-dev": {
+        "rector/rector": "^0.8.40"
+    },
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Abdul Malik Ikhsan",
+            "email": "samsonasik@gmail.com"
+        }
+    ],
+    "require": {}
+}
```

Comment:
Here should be "autoload" section too, ideally PSR-4, so it reflects real projects.

Reply:
It see there is no namespace bellow. Maybe then just "files" or "classmap".

The reason is that autoloading is the most common first problem people have while using Rector.
This would bring more focus on making autoloading correct first.

Reply:
updated in latest commit.